cdk/backend.ts

```diff
@@ -163,7 +163,22 @@ const coapDockerfilePath = path.join(
 	'containers',
 	'coap',
 )
-const coapHash = await hashFolder(coapDockerfilePath)
+const coapSimulatorDownloadUrl = (
+	await getCoAPHealthCheckSettings({
+		ssm,
+		stackName: STACK_NAME,
+	})
+).simulatorDownloadURL.toString()
+const { headers, ok, status } = await fetch(coapSimulatorDownloadUrl, {
+	method: 'HEAD',
+})
```

Comment:
This needs to be cached, so it does not download the file every time we run CDK.

Reply:
It does not download every time. Basically, I use `HEAD` to get file metadata. In S3, it provides `etag` in the header where it is never change if the file is the same.